src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,90 @@ Keywords:
 
 # Release notes
 
+## April 24, 2024
+
+### React Native Optimize 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native Places 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native Target 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native UserProfile 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native Messaging 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native EdgeIdentity 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native Edge 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native EdgeBridge 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native EdgeConsent 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native Campaign Standard 6.0.1
+
+* Added namespace support to the Android module.
+
+### React Native Assurance 6.0.1
+
+* Added namespace support to the Android module.
```

Comment:
I understand _why_ it renders like this, but I'm guessing there isn't an easy way to collapse these into a smaller heading?

Reply:
It is intended to display release notes in this format. If we want to hide them under a collapsible component, it may not be easy for customers to search for any React Native libraries.